content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -623,7 +626,7 @@ basis. The [Secrets](/docs/concepts/configuration/secret/#using-secrets-as-files
 
 ### Mounted ConfigMaps are updated automatically
 
-When a ConfigMap already being consumed in a volume is updated, projected keys are eventually updated as well. Kubelet is checking whether the mounted ConfigMap is fresh on every periodic sync. However, it is using its local ttl-based cache for getting the current value of the ConfigMap. As a result, the total delay from the moment when the ConfigMap is updated to the moment when new keys are projected to the pod can be as long as kubelet sync period (1 minute by default) + ttl of ConfigMaps cache (1 minute by default) in kubelet. You can trigger an immediate refresh by updating one of the pod's annotations.
+When a ConfigMap already being consumed in a volume is updated, projected keys are eventually updated as well. The kubelet is checking whether the mounted ConfigMap is fresh on every periodic sync. However, it is using its local TTL-based cache for getting the current value of the ConfigMap. As a result, the total delay from the moment when the ConfigMap is updated to the moment when new keys are projected to the pod can be as long as kubelet sync period (1 minute by default) + TTL of ConfigMaps cache (1 minute by default) in kubelet. You can trigger an immediate refresh by updating one of the Pod's annotations.
```

Comment:
nit: Would be good to reword this paragraph. OK to leave to another PR.
The kubelet checks whether the mounted ...